src/cypress-api/index.ts

```diff
@@ -0,0 +1,55 @@
+// @ts-nocheck
+
+import { writeTestResult } from '../write-archive';
+import { SourceMapper } from '../utils/source-mapper';
+
+const doArchive = async ({
+  testTitle,
+  domSnapshots,
+  resourceArchive,
+  chromaticStorybookParams,
+}) => {
+  let sourceMap: Map<string, string> | null = null;
+
+  if (domSnapshots.length > 0) {
+    // shortens file names in the last snapshot (which is the automatic one)
+    const sourceMapper: SourceMapper = new SourceMapper(domSnapshots[domSnapshots.length - 1]);
+    sourceMap = sourceMapper.shortenFileNamesLongerThan(250).build();
```

Comment:
This will likely be obsolete when the source-map-removing [code](https://github.com/chromaui/test-archiver/pull/28) is merged, will update here to reflect that.